components/camel-atmosphere-websocket/src/main/java/org/apache/camel/component/atmosphere/websocket/WebsocketEndpoint.java

```diff
@@ -86,15 +86,15 @@ public void setSendToAll(boolean sendToAll) {
         this.sendToAll = sendToAll;
     }
 
-    public boolean isUseStreaming() {
-        return useStreaming;
+    public boolean isStreaming() {
+        return streaming;
     }
 
     /**
      * To enable streaming to send data as multiple text fragments.
      */
-    public void setUseStreaming(boolean useStreaming) {
-        this.useStreaming = useStreaming;
+    public void setStreaming(boolean streaming) {
```

Comment:
I hope there is a better way, renaming a property should be avoided

Reply:
I've spent some time, and the only solution I found is renaming that property, I still cannot explain why, but, when useStreaming=true is used, all the messages are consumed by the same endpoint. https://issues.apache.org/jira/browse/CAMEL-18978

Reply:
TBH, this is still puzzling me

Reply:
Hi @essobedo thanks a lot for your review, you were right, useStreaming was not the issue, after some time debugging it, I noticed that Atmosphere builds an in-memory map for each consumer but, without this information https://github.com/apache/camel/pull/9210/files#diff-c20b148a113e2656ed364867af72660ee3204358f5ab036e1f4ac3c810f5c88fR62, the path, the logic in Atmosphere is not enough, and messages are redirected to the first entry of the map.

Reply:
Well done! Great news 🎆